mysql-test/suite/rocksdb/t/allow_to_start_after_corruption.test

```diff
@@ -60,7 +62,7 @@ select * from t1;
 # remove flag to ignore corruption
 --let $_mysqld_option=--log-error=$LOG
 --error 0
---exec $MYSQLD_CMD $_mysqld_option
+--exec $MYSQLD_CMD $_mysqld_option --default_dd_storage_engine=$ddse
```

Comment:
Here and elsewhere, why it's needed? Does it use the other DDSE otherwise?

Reply:
yes. if it isn't specified, it will use INNODB(since this is default DDSE). During restart, it found out the actual ddse is rocksdb, then try to convert from ROCKSDB DDSE to INNODB DDSE, which isn't supported yet.